index.d.ts

```diff
@@ -71,9 +71,10 @@ export interface AxiosRequestConfig {
   proxy?: AxiosProxyConfig | false;
   cancelToken?: CancelToken;
   decompress?: boolean;
+  lookup?: (hostname: string, options: any, callback: (err: NodeJS.ErrnoException, addr: string, family: 4 | 6) => void) => void
```

Comment:
The NodeJS namespace needs to be imported before being used, that is why CI is failing

Reply:
Thanks for the hint, now I just use the `Error` instead of `NodeJS.ErrnoException` and it seems fine.